apis/api_extension/v1alpha1/conversion/conversion_test.go

```diff
@@ -0,0 +1,66 @@
+/*
+Copyright 2015 The Kubernetes Authors.
```

Comment:
I would suggest renaming the `api_extension` folder as `visibility`. To keep the naming consistent between `apis/` and `pkg`.

Reply:
Fixed